backend/layers/common/entities.py

```diff
@@ -178,6 +178,11 @@ class DatasetMetadata:
     donor_id: List[str]
     is_primary_data: str
     x_approximate_distribution: Optional[str]
+    default_embedding: Optional[str] = None
+    embeddings: Optional[List[str]] = None
+    feature_biotype: Optional[List[str]] = None
+    feature_count: Optional[int] = None
+    feature_reference: Optional[List[str]] = None
```

Comment:
note: I'm realizing now that, without backfilling the previous DatasetVersions, these will need to stay as Optionals with None as the default arg otherwise we'll get errors fetching prior versions. Not sure if its worth considering a backfill plan as a follow-up (could we get away with just appending these attributes with a null field for all previous versions, so we don't need to reprocess old datasets?)

Reply:
We can skip back filling as long as they are optional. These will get added when we migrate anyways right? We can default to None during the migration if it doesn't exists.